webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/core_menu_options.tsx

```diff
@@ -0,0 +1,120 @@
+
+type Props = {
+    handleOnSelect: (e: React.FormEvent, scheduledAt: number) => void;
+}
+
+function CoreMenuOptions({handleOnSelect}: Props) {
+    const userTimezone = useSelector(getCurrentTimezone);
+
+    const today = moment().tz(userTimezone);
+    const tomorrow9amTime = moment().
+        tz(userTimezone).
+        add(1, 'days').
+        set({hour: 9, minute: 0, second: 0, millisecond: 0}).
+        valueOf();
+
+    const timeComponent = (
+        <Timestamp
+            value={tomorrow9amTime.valueOf()}
+            useDate={false}
+        />
+    );
+
+    const tomorrowClickHandler = useCallback((e) => handleOnSelect(e, tomorrow9amTime), [handleOnSelect, tomorrow9amTime]);
+
+    const optionTomorrow = (
+        <Menu.Item
+            key={'scheduling_time_tomorrow_9_am'}
+            onClick={tomorrowClickHandler}
+            labels={
+                <FormattedMessage
+                    id='create_post_button.option.schedule_message.options.tomorrow'
+                    defaultMessage='Tomorrow at {9amTime}'
+                    values={{'9amTime': timeComponent}}
+                />
+            }
+        />
+    );
+
+    const nextMonday = moment().
+        tz(userTimezone).
+        day(8). // next monday; 1 = Monday, 8 = next Monday
+        set({hour: 9, minute: 0, second: 0, millisecond: 0}). // 9 AM
+        valueOf();
+
+    const nextMondayClickHandler = useCallback((e) => handleOnSelect(e, nextMonday), [handleOnSelect, nextMonday]);
+
+    const optionNextMonday = (
+        <Menu.Item
+            key={'scheduling_time_next_monday_9_am'}
+            onClick={nextMondayClickHandler}
+            labels={
+                <FormattedMessage
+                    id='create_post_button.option.schedule_message.options.next_monday'
+                    defaultMessage='Next Monday at {9amTime}'
+                    values={{'9amTime': timeComponent}}
+                />
+            }
+        />
+    );
+
+    const optionMonday = (
+        <Menu.Item
+            key={'scheduling_time_monday_9_am'}
+            onClick={nextMondayClickHandler}
+            labels={
+                <FormattedMessage
+                    id='create_post_button.option.schedule_message.options.monday'
+                    defaultMessage='Monday at {9amTime}'
+                    values={{
+                        '9amTime': timeComponent,
+                    }}
+                />
+            }
+        />
+    );
+
+    let options: React.ReactElement[] = [];
+
+    switch (today.day()) {
+    // Sunday
+    case 0:
+        options = [optionTomorrow];
+        break;
+
+        // Monday
+    case 1:
+        options = [optionTomorrow, optionNextMonday];
+        break;
+
+        // Friday and Saturday
+    case 5:
+    case 6:
+        options = [optionMonday];
+        break;
+
+        // Tuesday to Thursday
+    default:
+        options = [optionTomorrow, optionMonday];
+    }
+
+    return (
+        <React.Fragment>
```

Comment:
There is no difference between the two not doing it.

Reply:
technically no, but for code readability, maintainability and follow new standard and guidelines there is a difference. I'm ok not changing it now, but for future reference only use `React.Fragment` if you need to set a key value, otherwise use the short version 👍